PROTOCOL.md

```diff
@@ -1933,7 +1933,9 @@ The request body should be a JSON string containing the following optional field
 
 - **timestamp** (type: String, optional): an optional timestamp string in the [Timestamp Format](#timestamp-format),. If set, will return files as of the table version corresponding to the specified timestamp. This is only supported on tables with history sharing enabled.
 
-- **startingVersion** (type: Long, optional): an optional version number. If set, will return all data change files since startingVersion, including historical metadata if seen in the delta log.
+- **startingVersion** (type: Long, optional): an optional version number. If set, will return all data change files since startingVersion, inclusive, including historical metadata if seen in the delta log.
+
+- **endingVersion** (type: Long, optional): an optional version number, only used if startingVersion is set. If set, will return all data change files until endingVersion, inclusive, including historical metadata if seen in the delta log. If this parameter is not supported by server, all data change files until the latest table version will be returned.
```

Comment:
Perhaps add a caution:
CAUTION: The endingVersion parameter may not be honored by earlier versions of the delta sharing server which may return all data change files until the latest table version. The clients should handle this case by filtering out the out of range files.

Reply:
sg.

Reply:
updated.